### PR TITLE
Yugaboard integration of pgx, node-postgres and rust-postgres

### PR DESCRIPTION
**Pull request by HarshDaryani896**
Each of the following tools was modified to generate a final test report in the following format:
```
{ "test_name": "load_balance", "script_name": "pgx/start.sh", "result": "PASSED", "error_stack": "" }
```
This is sent to the jenkins pipeline, which generates a payload for each test case based on this json. 

Tools changed:

- PGX
- node-postgres
- rust-postgres

Debug pipeline run: https://jenkins.dev.yugabyte.com/job/users/job/ecosystem-integration-debug/168/

### CODE REVIEW

File: node-postgres/run-app.sh [hunk 1]
```diff
@@ -25,6 +28,24 @@ else
...
+# Function to run individual test cases and capture their results
+run_test() {
+    local test_name=$1
+    local script_name=$2
+    echo "Running $test_name from $script_name..."
+
+    # Run the specific test case and capture errors
+    node "$test_name.js"  2>&1 | tee ${test_name}.log
+    if ! grep "Test Completed" ${test_name}.log; then
+      tail -n 30 ${test_name}.log | awk '{printf "%s\\n", $0}' > stack4json.log
```
Comment (ashetkar) [inline node-postgres/run-app.sh]: Any other string to search for instead of always taking last 30 lines?

File: node-postgres/run-app.sh [hunk 2]
```diff
@@ -36,88 +57,51 @@ echo "Exporting log level."
...
+run_test "yb-topology-aware-with-stop-node" "node-postgres/start.sh"
 
-grep "Test Completed" $ARTIFACTS_PATH/yb-load-balance-with-stop-node.txt
+# Finalize the JSON report
+sed -i '$ s/,$//' temp_report.json # Remove trailing comma from the last JSON object
+echo "]" >> temp_report.json
 
-grep "Test Completed" $ARTIFACTS_PATH/yb-pooling-with-load-balance.txt
+# Move the temporary report to the final report file
+mv temp_report.json "$REPORT_FILE"
```
Comment (ashetkar) [inline node-postgres/run-app.sh:105]: If the report contains tabs, those need to be replaced by spaces.

**Comment by ashetkar** [comment]
Also, please escape standalone backslash and double quotes in the `error_stack` field:
```
sed -i 's/\\\([^"'\'']\)/\\\\\1/g' stack4json.log  # Insert \ before another \ which is not followed by either " or '
sed -i 's/\"/\\\\\"/g' stack4json.log  # Replace " with \"
```

**Comment by ashetkar** [comment]
Please use the new script @Sfurti-yb added to format the json payload. Reference: https://github.com/yugabyte/ecosystem-test-automation/blob/test-automation-new/flyway/run-app.sh#L35

**Comment by HarshDaryani896** [comment]
Replaced by #30

**Pull request closed without merging**